topi/python/topi/cuda/fully_connected.py

```diff
@@ -0,0 +1,77 @@
+# pylint: disable=invalid-name
+"""Schedule for fully connected operators"""
+from __future__ import absolute_import as _abs
+import tvm
+
+def schedule_fully_connected(outs):
+    """Schedule for fully_connected.
+
+    Parameters
+    ----------
+    outs: Array of Tensor
+        The computation graph description of fully_connected
+        in the format of an array of tensors.
+
+    Returns
+    -------
+    s: Schedule
+        The computation schedule for fully_connected.
+    """
+    outs = [outs] if isinstance(outs, tvm.tensor.Tensor) else outs
+    s = tvm.create_schedule([x.op for x in outs])
+    Out = outs[0]
+
+    num_thread = 64
+    block_x = tvm.thread_axis("blockIdx.x")
+    block_y = tvm.thread_axis("blockIdx.y")
+    thread_x = tvm.thread_axis((0, num_thread), "threadIdx.x")
+
```

Comment:
add store_condition as in http://docs.tvmlang.org/tutorials/language/reduction.html#sphx-glr-tutorials-language-reduction-py